src/lightning_app/core/queues.py

```diff
@@ -317,14 +318,14 @@ def __init__(self, name: str, default_timeout: float):
         """
         if name is None:
             raise ValueError("You must specify a name for the queue")
-        self.app_id, self.name = self._split_app_id_and_queue_name(name)
-        self._original_name = name  # keeping the name for debugging
+        self.app_id, self._name_suffix = self._split_app_id_and_queue_name(name)
+        self.name = name  # keeping the name for debugging
         self.default_timeout = default_timeout
-        self.client = HTTPClient(base_url=HTTP_QUEUE_URL, log_callback=debug_log_callback)
+        self.client = HTTPClient(base_url=HTTP_QUEUE_URL, auth_token=HTTP_QUEUE_TOKEN, log_callback=debug_log_callback)
```

Comment:
Neat, for another PR, could you add the release version as a suffix ?

Reply:
It's in my TODOs. Will do